### ISSUE TITLE
HTTP parser should mention that it's based on httptools

### ISSUE THREAD

**Issue by 1st1** [vibora-io/vibora#37]
`vibora/vibora/parsers/parser.pyx` should mention that the code is copied from https://github.com/magicstack/httptools and thus bears MagicStack's copyright.

**Comment by frnkvieira** [comment]
You're completely right, any tips on this? Should I mention it only in this file? Do I need to mention it somewhere else? (I'm a complete beginner on this)

**Comment by 1st1** [comment]
Just adding 

```
# Portions Copyright (c) 2015-present MagicStack Inc.
# https://github.com/MagicStack/httptools
```

should suffice.  Thanks!

**Comment by 1st1** [comment]
BTW, why don't you just use httptools? If some functionality is missing we can probably add it.

**Comment by frnkvieira** [comment]
The main reason is that I need the .pxd files so I can link it with my own cython modules (not sure if there is another way to do this)... I also did some modifications in the API to speed up a little bit but that's specific to Vibora... your API is awesome and I don't think my modifications would fit other people use cases.
